Apps/WebClient/src/NewClientApp/src/components/site/FeedbackComponent.vue

```diff
@@ -0,0 +1,272 @@
+<script setup lang="ts">
+import { useVuelidate } from "@vuelidate/core";
+import { maxLength, minLength, required } from "@vuelidate/validators";
+import { computed, ref } from "vue";
+
+import HgButtonComponent from "@/components/common/HgButtonComponent.vue";
+import HgIconButtonComponent from "@/components/common/HgIconButtonComponent.vue";
+import { container } from "@/ioc/container";
+import { SERVICE_IDENTIFIER } from "@/ioc/identifier";
+import { ResultError } from "@/models/errors";
+import User from "@/models/user";
+import { IUserFeedbackService } from "@/services/interfaces";
+import { useErrorStore } from "@/stores/error";
+import { useUserStore } from "@/stores/user";
+
+defineExpose({
+    showDialog,
+});
+
+const userFeedbackService = container.get<IUserFeedbackService>(
+    SERVICE_IDENTIFIER.UserFeedbackService
+);
+
+const userStore = useUserStore();
+const errorStore = useErrorStore();
+
+const comment = ref("");
+const visible = ref(false);
+const hasSubmitted = ref(false);
+const isSuccess = ref(false);
+const isLoading = ref(false);
+
+const validations = computed(() => ({
+    comment: {
+        required,
+        minLength: minLength(5),
+        maxLength: maxLength(500),
+    },
+}));
+
+const v$ = useVuelidate(validations, { comment });
+
+const user = computed<User>(() => userStore.user);
+const hasEmail = computed<boolean>(
+    () => user.value.verifiedEmail && user.value.hasEmail
+);
+const isInvalid = computed<boolean>(
+    () => !v$.value.$dirty || (v$.value.$dirty && v$.value.$invalid)
+);
```

Comment:
```suggestion
const isInvalid = computed(() => ValidationUtil.isValid(v$.value) !== true);
```
This can be simplified with the ValidationUtil, however I'd suggest going further:

The property can be changed to `isValid` and return the result of `ValidationUtil.isValid()` directly.  This is how it is commonly used elsewhere.  Then places where `isInvalid.value` was used can check `isValid.value !== true` instead.